.github/workflows/release.yml

```diff
@@ -240,3 +242,28 @@ jobs:
           asset_path: javaagent/build/libs/opentelemetry-javaagent-${{ steps.set-versions.outputs.release-version }}.jar
           asset_name: opentelemetry-javaagent.jar
           asset_content_type: application/java-archive
+
+  create-pull-request-against-main:
+    needs: release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4
+
+      - name: Bump download link version
+        run: |
+          sed -Ei "s,https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/download/v[0-9]+.[0-9]+.[0-9]+/,https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/download/v${{ needs.release.outputs.release-version }}/," README.md
```

Comment:
Was wondering whether for main branch it doesn't make sense to just use `latest`

Reply:
I think that makes sense, it does revert the decision in #4774, but we've recently decided that `main` docs reflect SNAPSHOT state anyways (for some reason those things connect for me, even though I'm not sure they really do)